_maps/_basemap.dm

```diff
@@ -5,7 +5,6 @@
 #ifndef LOWMEMORYMODE
 	#ifdef ALL_MAPS
 		#include "map_files\Mining\Lavaland.dmm"
-		#include "map_files\generic\City_of_Cogs.dmm"
```

Comment:
Should stay here, this is used by Travis to make sure the map compiles.

Reply:
Doesn't travis just check everything including ruins now anyway? This is for local compiling I thought

Reply:
Travis's everything-check only includes the templates.

https://github.com/tgstation/tgstation/blob/e60014f7603029a1010c34c6073dc8fa45253e1c/tools/travis/template_dm_generator.py#L7-L10

Reply:
Should I just add it to that then? I don't think the average person testing if their funny hat spawns wants to spend the time compiling the city of cogs

Reply:
`ALL_MAPS` is usually only set by Travis.

It might make sense if the script just actually listed all maps, rather than listing some of them and requiring us to manually list the rest, but that doesn't need to be done now if you don't want to.